extensions/wikia/Forum/ForumController.class.php

```diff
@@ -85,8 +88,9 @@ protected function redirectToIndex() {
 	}
 
 	protected function getTopicTitle() {
-		$text = $this->wg->Title->getText();
-		$topicTitle = Title::newFromText( $text );
+		/** @var Title $title */
+		$title = $this->request->getVal( 'title' );
+		$topicTitle = Title::newFromText( $title->getText() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$topicTitle". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)